Gemfile

```diff
@@ -12,7 +12,8 @@ gem 'devise'
 gem 'pundit'
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 
-gem 'rails', '4.2.4'
+
+gem 'rails', '5.0.0.beta1.1'
```

Comment:
I would recommend downgrading to 4.2.5  

You should probably be ok to just change it and bundle install.  If it causes problems though you can go back to 5.